src/tests/data/pubkey/sphincsplus.vec

```diff
@@ -0,0 +1,90 @@
86A5509EC634E12B87CC22913A984429DAD57D7BA33969A7B2CE213C7BAE9CC641A403A0E9F9F1E6D43971A4D8E41BF3C9D756BE99A42EF46BBEC4FCB0EF951D12A63E1EF5CD8E4CCCA00EA75B66EEE04CE16F02ECEF1E321EC8EB9A1338944396843339EF065FF9DF68A11028907F45F906CD87350F1DB279D4B10D599B0BBE3CE1C5747FF3812EA5C242E76E67AE977F449263C8EA38DBACD48AF0913B27CD7F84B20B02BA6D50760281F6E85828E4D974CB0A73EFAB92A5330A22AD00C85504E5514392E8FA22E3E7AA3DF1BCD6D0D56AFBE20DF04D1908D7445F03D9A5B2234DB0C0C8A2924CECB2FB2B9A329040FE3B3CF52589E118EB75DE1C3A02596C0159CDFCF840CFA682B2548863D9AB34F3D82F73CBE78897E40ADBD96F9240E1B4A51A0D22E58DDDDA4DBBF90C1037F3B631777DD9F870B017B337697B38906F68F6299EFC5026E9C774903BC8A8A85A2F0FDE95DC7C20C1B4540C495569BD9D7539AE53D7E2DF5D214D1EAA85E7B551F7752E98401CA38337E21A9857E346B6CFDE9E80789A0515808246C610A2E9F8C288F24B7628ECC98DC6B2848B0098D0E44E1531C8676448CD2D393835EBED828B78E2AE820BA3D954590ADCE7E1C227259903796280312EB16F00B12D7C30F09A683884FA3B2DBAE9E47593F16ED7F3EF4704A7093404670646B3EEB062F9BE55AF5244B7F0315B273C3320CB9343E9251DD00EFA991FE9A34FB10C1DBB95215654DDE8AD0EBF922ECAAC3EA60D209DA466C9C99B86539909C331442FDF233BD8D2EA77AB9053D676C95D249DF72F20A8ADC25792AA9FACA94F65C0841D43A585099B2B36EB9B2BE76396CA5CD6BED0BB4357A5EE6F5385343B2A3D12CAEBDB836CEA569031A5793C38580E1575A5282E5F2B376C542C13B81FE1A1681714C635AA5E1F643C616131B8436958775C1FFBB3C8A02A8DBCD7175E4BE9D5A4AD252FF9A3D4A9A5E5554CE2E0C9D2F9C3EC2389EC2DEDBCB2B734FE8AFB95B5E83312BB220E35EF3E00CCA7935D0294DAC42946E09802DE574257CA7F4390D78F21360E3AB6B181590E87855EAD08448D4DF6CEF788E8822931BF9CF265F0A2EE8D029A3D1F8A4F795A9B60B77B52D2F551129468AE2B12764D7109E2CBBBA8EE837C2A3DBF2FD88D2154493E502B6BF7A0747814160B68B72DD1824FBAD318C2BA12FA16A5934F5A390F4ABA1808DC60FC5CDE8484BB431105AAB788588D243C18C33B05029B7C84C0D643600ED8C1D250E883BD524A83614859B0E025549B18C03D2894F37B9C584E0B371212D68BCFED60B18570B0CF63645C808286B9765725754E4F751DAF403815182B21C4B301183E2799F6BC144F16F98A96BAFFFC8B0B63B505762E2F66E9DCB44802D04D80CA8B33731D7B62DA025AB83B4430CEB4CDD0DE2433C396227E90F0DBD0C827826F9209A7672846B78BACDB935A271F3F18BE486952826C29EDD709D910EDCA1A8FB598712A3776DDECE903F247A88E4247A093B5DB0AA8FBC51572263F12063BDC8DA8140843F581DABF765FCFCE299CE315E67E7C965D51DABFC2AD5EB72426E0C43D60551E1C56170CA73F89B6782C4027B5ECBD7EA87D611E82C3BF54CCE971A60A719BE00E3C81B60E7B375C1532852CEF9DAFB88A130B218AC0F4C232C8418120E14EEF266C2299F640C73B09F70779771C226FA4C3B8B00D52A195E2671201379E5245D5DCE08DF4AB4CBB9C2478D9B96F32413453988C875299C5E4A315623DE8ACB3F4F3039D2D7395A424C7C998FE270B6A21214AC1BDE4730FD8E991DFF3D38120BDC8223003A5A524E04A2FB9BA7FFBC2570076BF2A378E7C8F97F2CA4C74C4AA2B6B2A8EAC571053B9553A8A96A6BD83BF665D473C742B6B69CFEBB8763AF518A716C180A41C2A66E6F5394CC8262779200C3279995A5D7452C25B864A9C8EA424E1334262871B2AF99859CF79F71A256EC6F580068359DBD704BA65C4926132F39B723BD145C88479D7154E5F561AA0919217174BE0204B0E85F8ECB3850ADA9412E69DD919873AE9D2849BA615EB46D7B0EE036F2DD81C4D8D734FCBFBEADE3D3F8A039FAA2A2C9957E835AD55B22E75BF57BB556AC8
```

Comment:
Yeah, there are pros and cons for your suggestion.
Pro: Obviously, much shorter and smaller file size. 
Con: The full signature helps for debugging, since the Sphincs+ signature is a concatenation of multiple parts. When searching for errors one could inspect which part of the signature is false to identify the respective part of the program.

IMO we should apply your suggestion anyway.

Reply:
Done